styles.css

```diff
@@ -0,0 +1,360 @@
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "roboto", sans-serif;
+}
+/************* navbar ******************/
+#navbar {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  height: 2%px;
+  background-color: #ffffff;
+  position: fixed;
+  width: 100%;
+  overflow: hidden;
+  top: 0;
+  z-index: 1;
+}
+#logo {
+  display: flex;
+}
+#logo img {
+  width: 27px;
+  justify-content: flex-start;
+  margin: 15px 0 15px 15px;
+}
+#logo h4 {
+  letter-spacing: -1.5px;
+  font-stretch: ultra-condensed;
+  padding-left: 1px;
+}
+.search-bar {
+  margin: auto;
+  display: flex;
+}
+.search-bar input {
+  border: 0.7px solid #cccccc;
+  border-radius: 1;
+  height: 25px;
+  width: 520px;
+  padding: 2px 23px 2px 10px;
+  outline: 0;
+  background-color: #fff;
+  font-size: 16px;
+  color: #858585;
+}
+button {
+  font-weight: 100;
+  color: #717171;
+
+  padding-right: 25px;
+  padding-left: 25px;
+}
+#navbar ul {
+  list-style-type: none;
+  margin: 0 30px 0 0;
+  padding: 0;
+  display: flex;
+}
+ul {
+  list-style-type: none;
+  padding: 0;
+  display: flex;
+}
+#navbar ul li {
+  margin: 20px 14px;
+  color: #606060;
+  display: flex;
+  font-size: 18px;
+}
+.bars {
+  margin: auto 10px auto 30px;
+}
+/************Nabar-end************/
+.main {
+  margin-top: 60px;
+  background-color: #f9f9f9;
+}
+.main-container {
+  display: flex;
+  margin-top: 10px;
+}
+/*************** video section ************/
+.video-section {
+  flex: 2;
+  margin: 25px 25px 25px 100px;
+}
+.video {
+  margin: 0px 0px 25px;
+}
+.under-video-section {
+  display: flex;
+  flex-direction: column;
+  margin: 15px;
+}
+.title {
+  margin: 10px 0 10px 0;
+  font-weight: 400;
+  font-size: 20px;
+}
+.channel-info {
+  display: flex;
+  justify-content: space-between;
+}
+.channel-icon img {
+  width: 80%;
+}
+.channel-info-start {
+  display: flex;
+}
+.channel-name {
+  display: block;
+  margin: 0;
+  top: 20px;
+}
+.channel-name h3 {
+  padding: 0px;
+  margin: 4px 0 0 0;
+  font-weight: 500;
+  font-size: 14px;
+}
+.channel-name .sub {
+  padding: 5px 4px 4px 4px;
+  margin: 0;
+  font-size: 10px;
+  margin-top: 20px;
+  background-color: rgb(226, 0, 0);
+  border: none;
+  outline: none;
+  border-radius: 3px;
+  color: #eeeeee;
+}
+.channel-info-end {
+  margin-top: 5%;
+}
+.views {
+  font-size: 25px;
+}
+hr {
+  border: solid 0.7px #e6e6e6;
+  margin-top: 0;
+}
+.buttons {
+  display: flex;
+  justify-content: space-between;
+}
+.buttons-start {
+  display: inline-block;
+  justify-content: start;
+  padding-left: 0px !important;
```

Comment:
Try not to use `!important` in your code. If you need to override something you can do so using selector specificity.  You can learn more about that in [this article](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity) and in [this article](https://www.w3schools.com/css/css_specificity.asp)